src/validate.js

```diff
@@ -4,7 +4,11 @@ const ajv = new Ajv({
   errorDataPath: "property",
   allErrors: true,
   multipleOfPrecision: 8,
+  schemaId: "auto",
 });
+//flag indicating whether we've already added custom schemas
+let addedAdditionalSchemas = false;
```

Comment:
Why would we be disallowed from adding schemas multiple times ?

I could see a case where you would have schema-06 for one form, and schema-04 for an other form.

Reply:
@glasserc You can't add the same schema multiple times, ajv throws errors: 
![image](https://user-images.githubusercontent.com/17271979/51590017-ea14fa00-1ee8-11e9-8849-71c309342d66.png)

If you want to you can add multiple schemas at once, like so `additionalSchema ={[ schema1, schema2 ]}`

And if you want to validate one form using draft-6 and one using draft-4 you'd just use correct `$schema` in your schema, like in example provided in my second comment, read this for clarification https://github.com/epoberezkin/ajv#validateschemaobject-schema---boolean .

> If schema has $schema property, then the schema with this id (that should be previously added) is used to validate passed schema.
also  
>By default this method is called automatically when the schema is added, so you rarely need to use it directly.

@epicfaace They're additional jsonSchemas :P variable name can be changed, no problem

Reply:
Ok makes sense !